www/artists/wmulcaster/index.html

```diff
@@ -10,7 +10,7 @@ <h1>Wynona Mulcaster</h1>
 		<h3>EDUCATION</h3>
 		<div style="float:left; width:300px; border-style:solid; border-color:#000000">
 			<img style="float:left; width:300px" src="wynona_picture.jpg" alt="Picture of Wynona Mulcaster"/>
-			<p style="float:left; width:300px;">Okema beach, Emma Lake, December 1939. Left to right: Nonie Mulcaster, Dr. Leslie Saunders, Bodil Lindner,Ernest Lindner, Edith Cook (?)</p>
+			<p style="float:left; width:300px;">Okema beach, <a href="https://www.saskartists.ca/artists/emmalake/index.html">Emma Lake</a>, December 1939. Left to right: Nonie Mulcaster, Dr. Leslie Saunders, Bodil Lindner,<a href="https://www.saskartists.ca/artists/elindner/index.html">Ernest Lindner</a>, Edith Cook (?)</p>
```

Comment:
We can't allow absolute links to SaskArtists. This is to avoid potential issues if our domain changes.